spec/helpers/spec-helper.coffee

```diff
@@ -33,40 +45,48 @@ beforeEach ->
   $.fx.off = true
 
   # Basic page fixture
-  $('#jasmine_content').html("<div id='wrapper'></div><div id='overlays'></div><div id='side-nav'></div><div id='main-view'></div></div>")
-
-  # Setup a new base.
-  window.base = {}
-  window.base.data = new Brainstem.StorageManager()
-  window.base.data.addCollection 'time_entries', App.Collections.TimeEntries
-  window.base.data.addCollection 'posts', App.Collections.Posts
-  window.base.data.addCollection 'tasks', App.Collections.Tasks
-  window.base.data.addCollection 'projects', App.Collections.Projects
-  window.base.data.addCollection 'users', App.Collections.Users
-
+  $(document.body).html('''
+    <div id="jasmine_content">
+      <div id="wrapper"></div>
+      <div id="overlays"></div>
+      <div id="side-nav"></div>
+      <div id="main-view"></div>
+    </div>
+  ''')
+
+  # Instantiate storage manager
+  storageManager = StorageManager.get()
+  storageManager.addCollection 'time_entries', TimeEntries
+  storageManager.addCollection 'posts', Posts
+  storageManager.addCollection 'tasks', Tasks
+  storageManager.addCollection 'projects', Projects
+  storageManager.addCollection 'users', Users
```

Comment:
It would be nice if we could do this in a single call where we pass these in as key/value pairs. Not a blocker to get this out the door though.